sourcemap/src/lib.rs

```diff
@@ -114,46 +115,60 @@ impl SourceMapView {
     /// If the sourcemap is an index it is being flattened.  If flattening
     /// is not possible then an error is raised.
     pub fn from_json_slice(buffer: &[u8]) -> Result<Self, ParseSourceMapError> {
+        use sourcemap::DecodedMap::*;
```

Comment:
Please do not import enum variants. Even though you're matching only on tuple variants in this case, having such imports is a common cause of issues with typos.